app/src/main/java/io/reark/rxgithubapp/data/stores/StoreBase.java

```diff
@@ -0,0 +1,27 @@
+package io.reark.rxgithubapp.data.stores;
+
+import android.content.ContentResolver;
+import android.support.annotation.NonNull;
+
+import com.google.gson.Gson;
+
+import javax.inject.Inject;
+
+import io.reark.reark.data.store.SingleItemContentProviderStore;
+
+/**
+ * Created by antti on 17.1.2016.
+ */
+public abstract class StoreBase<T, U> extends SingleItemContentProviderStore<T, U> {
+
+    @Inject
+    Gson gson;
```

Comment:
Why don't we inject the objects mostly via the constructor instead. It is much cleaner and doesn't require coupling with graph